src/Microsoft.DotNet.Darc/Darc/Options/VirtualMonoRepo/CodeFlowCommandLineOptions.cs

```diff
@@ -37,9 +36,6 @@ internal abstract class CodeFlowCommandLineOptions<T> : VmrCommandLineOptions<T>
     [Option("build", Required = false, HelpText = "If specified, flows the given build. Cannot be used with --ref.")]
     public int? Build { get; set; }
 
-    [Option("commit", Required = false, HelpText = "If specified, flows the given commit. Cannot be used with --build.")]
```

Comment:
maybe it'd be useful if we kept the option to flow a build by specifying `repo` and `commit`, finding it in bar (like `get-build` has the option to do), and going forward with it

Reply:
I think we just don't know what will be useful for local dev yet so I think it would be premature to support the tooling. Or at least I see at least 3 different options that we could support but no idea if people would even use it.

Reply:
Let's disregard `darc vmr *-flow` commands for now